cfme/v2v/migrations.py

```diff
@@ -363,17 +371,51 @@ def plan_in_progress(self, plan_name):
             return False
 
 
+class MigrationDashboardView59z(MigrationDashboardView):
```

Comment:
Dashboard for 59z has `infra_mapping_list` while 510z moves it to a separate page.